documentation/docs/liquidity-mining/howto.md

```diff
@@ -33,6 +45,9 @@ Check your performance in the real-time Activity section within our liquidity mi
 
 Once the program officially kicks off, the rewards will be paid out **weekly** in USDC based on the **actual trading activity** of the participants. 
 
+!!! Tip
+    The rewards will be automatically calculated based on your trading activities for eligible pairs collected through the exchange APIs you provide. Please check out the individual campaign page for the eligible pair information. 
```

Comment:
"Please check each individual campaign page for specific terms, conditions, and eligibility information."